### DISCUSSION TITLE
Brass Golems

### DISCUSSION THREAD

**Discussion by DautApsua** [ParadiseSS13/Paradise#22228] [Documents]
So i was thinking that it will be very funny and cool if **Brass Golems** were added to the server.

I have read some information about the brass itself and i think Brass Golems would be very cool to have.

So since the only way to get brass in the game is to go to lavaland (atleast i think so)  and get the brass from the old ruins, i think making the brass golem shell itself is very hard and is gonna be rewarding in the end because of its stats.

The brass itself is yellow, so obviously the Golem will be yellow too.

![32px-Gold_golem](https://github.com/ParadiseSS13/Paradise/assets/143400715/67f36ac8-a9a6-49a7-bc9a-6dac959855b3)

For the traits (since getting the brass itself is really hard) It can be something like this;

1.High burn resistant (probably same as plasititanium golems)/No burn damage at all (except if step on lava) 

2.Cant feel hot/Shock resistant

3.Resistance to lavaland ash storms

4.A bit faster than regular golems/Same speed(Since brass isnt very heavy metal

5.Cant get sick (brass is anti-bacterial metal so yeah its cool)

The toughness, damage dealt, stun resistance, etc is still the same as regular golems.

Brass doesnt really have any **disatvantages** but its still fine since you can really only spawn 1 Brass Golem (Depends if you get lucky in finding the ancient brass ruins in lavaland, which isnt that usual)

Feel free to comment your funnies111111

**Comment by DGamerL** [comment]
You can easily make brass using chemistry sadly, which if golems actually do their RnD job, they can easily obtain

**Reply by DautApsua** [reply]
Just realised, but still the idea is cool even though if its that easy to obtain.....the buffs arent that great then. Perhaps some debuffs?

**Reply by DautApsua** [reply]
Im gonna change the whole post later to make it a bit fair

**Comment by DGamerL** [comment]
👍 sounds good!

**Reply by DGamerL** [reply]
Wrong place to send this message, but we ball
